src/index.html

```diff
@@ -1,14 +1,43 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" 
+          href="https://fonts.gstatic.com" 
+          crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <div class="header-container">
+        <a href="#home">
+          <img src="images/logo.png" 
+          alt="images" 
+          class="img">
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__list-item nav__list-item-active"><a href="#apple" class="is-active">Apple</a></li>
```

Comment:
```suggestion
            <li class="nav__list-item nav__list-item-active">
              <a href="#apple" class="is-active">Apple</a>
            </li>
```
Новый тег - новая линия